src/api/java/baritone/api/Settings.java

```diff
@@ -73,6 +73,12 @@ public final class Settings {
      */
     public final Setting<Boolean> allowInventory = new Setting<>(false);
 
+    /**
+     * Allow Baritone to automatically put useful items (such as tools and throwaway blocks) on the hotbar while
+     * pathing. Having this setting enabled implies {@link #allowInventory}.
+     */
+    public final Setting<Boolean> allowHotbarManagement = new Setting<>(false);
```

Comment:
Baritone settings style so far was settings depending on each other rather than implying each other (`allowParkour` -> `allowParkourPlace`, `avoidance` -> `mobAvoidance`, maybe more) and from testing `blockFreeLook` I found it very annoying and ended up always having `freeLook` disabled and controlling it via `blockFreeLook` because that's less typing.

Reply:
Fair point, I did too encounter this issue while testing but never bothered to fix it. I've updated this setting accordingly and will go ahead with fixing `blockFreeLook` on `master` too.